app/src/main/java/com/nutomic/syncthingandroid/activities/SettingsActivity.java

```diff
@@ -186,9 +188,10 @@ public void onActivityCreated(Bundle savedInstanceState) {
             Preference environmentVariables = findPreference("environment_variables");
             Preference stReset              = findPreference("streset");
 
-            mUseRoot                     = (CheckBoxPreference) findPreference(Constants.PREF_USE_ROOT);
-            Preference useWakelock       = findPreference(Constants.PREF_USE_WAKE_LOCK);
-            Preference useTor            = findPreference("use_tor");
+            mUseRoot                        = (CheckBoxPreference) findPreference(Constants.PREF_USE_ROOT);
+            Preference useWakelock          = findPreference(Constants.PREF_USE_WAKE_LOCK);
+            Preference useTor               = findPreference("use_tor");
```

Comment:
I know this is not your change, but should we have a constant for this?

Reply:
Right, this would apply to many prefs. Looking through the code, only some are implemented with a constant. I would prefer doing the individual string to constant improvement in a new PR.

Reply:
I replaced this one with a constant on the way.